apps/remote_control/test/lexical/remote_control/build_test.exs

```diff
@@ -592,4 +615,8 @@ defmodule Lexical.BuildTest do
       assert loaded?(project, WithAType)
     end
   end
+
+  defp version_after_1_15? do
+    Version.match?(System.version(), "~> 1.15")
```

Comment:
this should be `>= 1.15.3` right?

Reply:
yes

Reply:
`~> 1.15` and `>= 1.15.3` are different.